main.go

```diff
@@ -8,7 +8,20 @@ import (
 	"github.com/deis/object-storage-cli/config"
 )
 
+var (
+	debugStrs = []string{"DEIS_DEBUG"}
+)
+
 func main() {
+	var debug bool
+	for _, str := range debugStrs {
```

Comment:
Since there's just `DEIS_DEBUG` now, should the loop be refactored out?
```go
valStr := os.Getenv("DEIS_DEBUG")
debug = (valStr == "true" || valStr == "1")
```